nodes.py

```diff
@@ -392,6 +393,7 @@ def INPUT_TYPES(cls):
                 "optimize_png":          ("BOOLEAN", {"default": False,                                            "tooltip": "if True, saved PNG files will be optimized (can reduce file size but is slower)"}),
                 "counter":               ("INT",     {"default": 0, "min": 0, "max": 0xffffffffffffffff,           "tooltip": "counter"}),
                 "denoise":               ("FLOAT",   {"default": 1.0, "min": 0.0, "max": 1.0,                      "tooltip": "denoise value"}),
+                "clip_skip":             ("INT",     {"default": -1, "min": -24, "max": -1,                        "tooltip": "clip skip value (positive int)"}),
```

Comment:
```suggestion
                "clip_skip":             ("INT",     {"default": -1, "min": -24, "max": -1,                        "tooltip": "clip skip value"}),
```
Can skip this part, since it supports both.